evap/staff/tests/test_views.py

```diff
@@ -1722,6 +1724,25 @@ def test_import_valid_participants_file(self):
         page = self.app.get(self.url, user=self.manager)
         self.assertNotContains(page, 'Import previously uploaded file')
 
+    def test_replace_valid_participants_file(self):
+        page = self.app.get(self.url2, user=self.manager)
+
+        original_participant_count = self.evaluation2.participants.count()
+
+        form = page.forms["participant-import-form"]
+        form["excel_file"] = (self.filename_valid,)
+        page = form.submit(name="operation", value="test-participants")
+
+        self.assertContains(page, 'Import previously uploaded file')
+        self.assertEqual(self.evaluation2.participants.count(), original_participant_count)
+
+        form = page.forms["participant-import-form"]
+        form.submit(name="operation", value="import-replace-participants")
+        self.assertEqual(self.evaluation2.participants.count(), 2)
```

Comment:
This assertion only works because the test setup currently ensures that they the participant count was unequal to two before. However, in the future, anyone might change the test setup. To prevent this from making the test meaningless, I like to add an assertion that the old state was differend. In this case, that would be:
```
self.assertNotEqual(self.evaluation2.participants.count(), 2)
```
before the submission of the form.

(same for the other tests)